packages/turf-voronoi/index.js

```diff
@@ -0,0 +1,45 @@
+import lineStringToPolygon from '@turf/linestring-to-polygon';
+import { lineString } from '@turf/helpers';
+import d3voronoi from 'd3-voronoi';
+
+function und3ify(polygon) {
+    return lineStringToPolygon(lineString(polygon.slice(0)));
+}
+
+/**
+ * Takes a FeatureCollection of points, and a bounding box, and returns a FeatureCollection
+ * of Voronoi polygons. 
+ * 
+ * The Voronoi algorithim used comes from the d3-voronoi package.
+ *
+ * @name voronoi
+ * @param {FeatureCollection<Point>|Feature[]<Point>} points to find the Voronoi polygons around.
+ * @param {number[]} bbox clipping rectangle, in [minX, minY, maxX, MaxY] order.
+ * @returns {FeatureCollection<Polygon} a set of polygons, one per input polygon.
+ * @example
+ * var bbox = [-70, 40, -60, 60];
+ * var points = turf.random('points', 100, { bbox: bbox }); 
+ * var voronoiPolygons = turf.voronoi(points, bbox);
+ */
+function voronoi(points, bbox) {
+    if (!points) {
```

Comment:
@stevage maybe we should instead throw an error here (since the parameter is required) or at least return "something", so it's clear that the input was empty.

Reply:
Agreed, both `points` & `bbox` should return Errors if not provided.

```js
if (!points) throw new Error('points is required');
if (!bbox) throw new Error('bbox is required');
```